extensions/extensions-helm/aissemble-configuration-store-chart/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: aissemble-configuration-store-chart
```

Comment:
Q/A: Do we think we should go ahead and add this to the `containers.adoc` page?  I know the feature isn't technically ready yet, but this feels like one of those things we'll overlook if we don't do it when we add the chart.

Reply:
I was thinking we would create one for the release its going into so people dont try and use it thinking its ready. If you dont think thats an issue then I see no problem adding it

Reply:
Yeah I could see the argument either way. Just worried we'll forget 😅

Reply:
Ill go ahead and add it to the "Available V2 Profiles"